packages/core/package.json

```diff
@@ -32,6 +32,7 @@
     "@backstage/config": "^0.1.1-alpha.23",
     "@backstage/core-api": "^0.1.1-alpha.23",
     "@backstage/theme": "^0.1.1-alpha.23",
+    "@backstage/plugin-catalog": "^0.1.1-alpha.23",
```

Comment:
We are using the `useEntity()` hook from `plugin-catalog`. Unsure if that's "allowed" as it adds this dependency. Any comments on that?

Reply:
Indeed, can't add this depedency here